### ISSUE TITLE
do you have files with real data and code this data ? 

### ISSUE THREAD

**Issue by Sandy4321** [py-why/EconML#811]
do you have files with real data and code this data ? 

i see you use synthetic data
https://github.com/py-why/EconML/blob/main/notebooks/Double%20Machine%20Learning%20Examples.ipynb

**Comment by kbattocchi** [comment]
@Sandy4321 No, we typically use synthetic data because even if we had real data, without somehow knowing the ground truth causal effect there would be no way to demonstrate that the correct results are recovered.
